<launch>

   <arg name="rgb_topic" default="/rgb/image_raw/compressed"/>
   <arg name="depth_topic" default="/depth/image_raw"/>
   <arg name="camera_info_topic" default="/depth/camera_info"/>
   <arg name="pose_topic" default="/pose"/>
   <arg name="publish_debug" default="false"/>
   <arg name="base_link_name" default="/base_link"/>
   <arg name="camera_link_name" default="/base_link"/>


   <!-- drone detection -->
   <node pkg="drone_detector_tracking" type="drone_detector_tracking_node" name="drone_detector" output="screen">
      <param name="rgb_img_topic" type="str" value="$(arg rgb_topic)"/>
      <param name="depth_img_topic" type="str" value="$(arg depth_topic)"/>
      <param name="camera_info_topic" type="str" value="$(arg camera_info_topic)"/>
      <param name="uav_pose_topic" type="str" value="$(arg pose_topic)"/>
      <param name="publish_debug_images" type="bool" value="$(arg publish_debug)"/>
      <param name="base_link_name" type="str" value="$(arg base_link_name)"/>
      <param name="camera_link_name" type="str" value="$(arg camera_link_name)"/>
   </node>


</launch>
